### DISCUSSION TITLE
Fans always locked to MAX speed in every profile

### DISCUSSION THREAD

**Discussion by Deflektor14** [seerge/g-helper#1787] [Q&A]
Hi, i have Asus Rog s15 gx502gw laptop, here is the problem. In every profile that i choose the fans always spins close to its max speeds. For example in balanced the fan speeds are always around 4000-4400 at 65-70 degrees temps. But in factory defaults profile at that temp it should spins at 3000-3200 rpm, 4000-4000 should be only at 98 degrees.

The only way i could lower rpm is by cheking the "apply custom fan curve" and tweaking the highest dot on the graph, basically limiting the max rpm speed. So the problem is that the whole curve is not working, its appears to be locked to the highest setting which is not optimal. Please help.

**Comment by seerge** [comment]
> **[How does G-helper control my fan speeds?](https://github.com/seerge/g-helper?tab=readme-ov-file#how-does-g-helper-control-my-fan-speeds)**
> It doesn't. Your firmware / BIOS controls them in real-time. Armoury also doesn't control fans in real time anyhow.

> What G-helper can do - is (optionally) set a custom fan profile to the current performance mode consisting of 8 pairs of temperature + fan speed % via the same endpoint Armoury seems to use. How it will be interpreted - is still up to the firmware.

**Reply by Deflektor14** [reply]
I get it, but it seems only the last 8 (highest) dot is recognized by the system. When I change it the fans speed go higher/lower, but the other 7 dots below it basically do nothing no matter how you change them.
